Sources/Crypto.h

```diff
@@ -53,6 +53,9 @@
 /// Computes the SHA256 hash of the SHA256 hash of the data.
 + (nonnull NSData *)sha256sha256:(nonnull NSData *)data;
 
+/// Computes the SHA256 hash of the data.
++ (nonnull NSData *)sha256:(nonnull NSData *)data;
```

Comment:
there is also `hash()` which is the same thing

Reply:
@vikmeup not exactly the same, `hash()` is too general, how about rename to `keccak256` or something similar